src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,59 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier implements NumberSupplier {
+    private static final int RADIUS_CIRCLE = 10;
+    private static final String DEFAULT_COLOR = Color.BLACK.toString();
+    private static final int AMOUNT_OF_FIGURE_TYPE = 5;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    
+    public Figure getRandomFigure() {
+        switch (random.nextInt(AMOUNT_OF_FIGURE_TYPE)) {
+            case 0:
+                return getRandomCircle();
+            case 1:
+                return getRandomIsoscelesTrapezoid();
+            case 2:
+                return getRandomRectangle();
+            case 3:
+                return getRandomRightTriangle();
+            case 4:
+            default:
+                return getRandomSquare();
+        }
+    }
+    
+    public int getRandomIntNumber() {
+        return random.nextInt(20);
```

Comment:
What is 20? Magic number :)
Make it constant field.

Also, you have special separate method for getting random number.
Why wouldn't to create the same, but for the color?